fastlane_bot/bot.py

```diff
@@ -133,9 +125,6 @@ def __post_init__(self):
 
         if self.TxRouteHandlerClass is None:
             self.TxRouteHandlerClass = TxRouteHandler
-        assert issubclass(
```

Comment:
why do you remove this assertion?

Reply:
Because I've removed the (empty) class `TxRouteHandlerBase`